roles/uperf-bench/library/k8s_log.py

```diff
@@ -0,0 +1,128 @@
+#!/usr/bin/python
```

Comment:
This file needs to be deleted.

Reply:
That got pulled back in during one of my bad merges. I'll clean it up.